models/reservation.js

```diff
@@ -152,6 +98,21 @@ exports.finish = function(reservationId) {
     });
 };
 
+exports.review = function(reservationId, rating, comment) {
+  var reservation;
+  return exports.updateStatus(reservationId, exports.ReservationStatusEnum.reviewed)
+    .then(function(res) {
+      reservation = res;
+      return Promise.all([
+        Spot.review(reservation.attributes.spotId, reservationId, rating, comment),
+        User.removeReservationFromDriverActive(reservation)
+      ]);
+    })
+    .then(function() {
+      return reservation;
```

Comment:
ya updateStatus calls get() at the end of it, so I use it at the start of all my status changing functions. Probably not the greatest code practice. I have an issue #35 that tries to make all inputs/outputs actual reservations for consistency. Except of course for any of the public functions, which should take in an ID